api/envoy/api/v2/route/route.proto

```diff
@@ -522,6 +522,9 @@ message RouteAction {
     // point the host that was last selected will be routed to. If unspecified, this will default to
     // retrying once.
     int64 host_selection_retry_max_attempts = 6;
+
+    // Http status codes that should trigger a retry in addition to those specified by retry_on.
```

Comment:
Nit: s/Http/HTTP/

Reply:
Also, can you add the ref for `retry_on` to the anchor (as done in `router_filter.rst` already).